### PR TITLE
Scan file using events

### PR DESCRIPTION
**Pull request by sharathkumaranbu**
Changes related to https://github.com/topcoder-platform/av-scanner-service/issues/8

### CODE REVIEW

File: src/services/ProcessorService.js [hunk 1]
```diff
@@ -44,16 +42,44 @@ function * processMessage (message) {
...
-  // scan the file in DMZ
-  logger.info(`Scanning the file ${fileName}.`)
-  const form = new FormData()
-  form.append('file', dmzS3Obj.Body, { filename: fileName })
-  const scanResult = yield axios.post(config.ANTIVIRUS_API_URL, form, { maxContentLength: config.MAXFILESIZE, headers: form.getHeaders() })
+  // Send request to Scan the file
+  logger.info(`Sending request to scan the file ${fileName}.`)
+  const reqBody = {
+    submissionId: message.payload.id,
+    url: message.payload.url,
```
Comment (callmekatootie) [inline src/services/ProcessorService.js]: @sharathkumaranbu The `url` cannot be passed from the message object here. It has to be the S3 url in DMZ storage only and thus, we cannot rely on the url from message object.

**Review by callmekatootie — CHANGES_REQUESTED** [review]

File: src/services/ProcessorService.js (hunk 1 above)
Comment (sharathkumaranbu) [reply src/services/ProcessorService.js]: @callmekatootie Fixed

**Comment by sharathkumaranbu** [comment]
@callmekatootie Requested changes have been done

**Review by callmekatootie — APPROVED** [review]

**Comment by callmekatootie** [comment]
@BadalZ This one is good to be merged in

**Pull request merged**